cloud_experts_tutorials/cloud-experts-rosa-hcp-activation-and-account-linking-tutorial.adoc

```diff
@@ -62,24 +71,39 @@ The ELB service-linked role is created for you automatically. You can click any
 
 . Click the orange *Continue to Red{nbsp}Hat* button to proceed with account linking:
 +
+.Continue to Red{nbsp}Hat
++
 image::rosa-continue-rh-6.png[]
 
 . If you are not already logged in to your Red{nbsp}Hat account in your current browser's session, you will be asked to log in to your account:
 +
+[NOTE]
+====
+Your AWS account must linked to a single Red{nbsp}Hat organization. 
+====
++
+.Log in to your Red{nbsp}Hat account
++
 image::rosa-login-rh-account-7.png[]
 +
 * You can also register for a new Red{nbsp}Hat account or reset your password on this page.
 * Make sure to log in to the Red{nbsp}Hat account that you plan to associate with the AWS account where you have activated {hcp-title} in previous steps.
-* Only a single AWS account that will be used for service billing can be associated with a Red{nbsp}Hat account. Typically an organizational AWS account that has other AWS accounts, such as developer accounts, linked would be the one that is to be billed, rather than individual AWS end user accounts.
-* Red{nbsp}Hat accounts belonging to the same Red{nbsp}Hat organization will be linked with the same AWS account. Therefore, you can manage who has access to creating {hcp-title} clusters on the Red{nbsp}Hat organization account level.
+* An AWS account that is used for service billing can only be associated with a single Red{nbsp}Hat account. Typically an organizational AWS account that has other AWS accounts, such as developer accounts, linked would be the one that is to be billed, rather than individual AWS user accounts.
```

Comment:
```suggestion
* The AWS account used for service billing can only be associated with a single Red{nbsp}Hat account. Typically a main AWS account that has other AWS linked accounts, such as a developer account, rather than individual AWS user accounts is the AWS account used for service billing.
```